doc/source/development/contributing.rst

```diff
@@ -254,6 +257,29 @@ You'll need to have at least python3.5 installed on your system.
    python setup.py build_ext --inplace -j 4
    python -m pip install -e . --no-build-isolation
 
+**Windows**
+
+Below is a brief overview on how to set-up a virtual environment with Powershell
+under Windows. For details please refer to the \
+`official virtualenv user guide <https://virtualenv.pypa.io/en/stable/userguide/#activate-script>`__
+
+.. code-block:: none
```

Comment:
I think using `bash` instead of `none` should highlight this in a nice way (comments in a different color). I know you copied it from the previous block, but I think you can change that too.

Reply:
Added bash to the previous block, powershell to the Windows one.